src/main/python/rerank/scripts/utils.py

```diff
@@ -0,0 +1,122 @@
+import shlex
+import subprocess
+import sys
+import string
+printable = set(string.printable)
+printable.remove("\n")
+printable.remove("\t")
+printable.remove("\r")
+import os
+
+import jnius_config
+jnius_config.set_classpath("target/anserini-0.4.1-SNAPSHOT-fatjar.jar")
+from jnius import autoclass
+JString = autoclass('java.lang.String')
+JSearcher = autoclass('io.anserini.search.SimpleSearcher')
+
+def cal_score(fn_qrels="src/main/resources/topics-and-qrels/qrels.microblog2014.txt", prediction="score.txt"):
+    cmd = "/bin/sh run_eval_new.sh {} {}".format(prediction, fn_qrels)
+    pargs = shlex.split(cmd)
+    p = subprocess.Popen(pargs, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    pout, perr = p.communicate()
+    print("running {}".format(cmd))
+    if sys.version_info[0] < 3:
+        lines = pout.split('\n')
+    else:
+        lines = pout.split(b'\n')
+    Map = float(lines[0].strip().split()[-1])
```

Comment:
I think `MAP` and `MRR` are fine here...

Reply:
Okay.